app/views/welcome/index.html.erb

```diff
@@ -1,3 +1,41 @@
+<body>
+
+<script>
+function myMove() {
+  var elem = document.getElementById("planeImage");   
+  var pos = 125; //start position of object upon click (relative to left side of container)
+  var id = setInterval(frame, 10); //will call frame method every 10ms
+  function frame() {
+    if (pos == 520) { //800 is distance it will travel
+      elem.className="rotated"
+      clearInterval(id);//will stop the setInterval method.
+      return myMove_left();
+
+    } else {
+      pos++; //increment position by 1px
+      elem.style.left = pos + 'px';  //pixel distance from left side of parent
+    }
+  }
+}
+
+
+function myMove_left() {
+  var elem = document.getElementById("planeImage");   
+  var pos = 520; //start position of object upon click (relative to left side of container)
+  var id = setInterval(frame_left, 10); //10 is speed of object, 1 is fast
+  function frame_left() {
+    if (pos == 125) { //800 is distance it will travel
+      elem.classList.remove("rotated");
+      clearInterval(id);
+      return myMove();
+    } else {
+      pos--; //increment position by 1px
+      elem.style.left = pos + 'px';  //pixel distance from left side of parent
+    }
+  }
+}
+</script>
+
 <!-- <div class="main-container"> -->
```

Comment:
I think this and the closing comment were leftover comments that can be deleted.